packages/instantsearch.js/src/connectors/refinement-list/connectRefinementList.ts

```diff
@@ -475,10 +475,6 @@ const connectRefinementList: RefinementListConnector =
               ? searchParameters.getDisjunctiveRefinements(attribute)
               : searchParameters.getConjunctiveRefinements(attribute);
 
-          if (!values.length) {
-            return uiState;
-          }
```

Comment:
there's similar code as this in every refinement widget, all needs to be replaced